wmr-api/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "wmr-api",
```

Comment:
See if we can add our logo as the icon for the application. Just to make it look nicer in the UI